census/estimator/README.md

```diff
@@ -0,0 +1,135 @@
+# Census: TF.Learn Distributed Training and Prediction
+
+This sample walks you through training and prediction, including distributed training, and hyperparameter tuning.
+
+All commands below assume you have cloned this repository and changed to the currend directory. It also assumes you have:
+
+1. Installed the the [Google Cloud SDK](https://cloud.google.com/sdk)
+2. (For Cloud Training) [Set up the Google Cloud Machine Learning API](https://cloud.google.com/ml/docs/how-tos/getting-set-up) NOTE: You can safely ignore the "Setting Up Your Environment" section. 
+
```

Comment:
Probably just want to point people to the quickstart for cloud ml which includes additional instructions (such as installing tensorflow)